orchagent/dash/dashvnetorch.cpp

```diff
@@ -490,14 +494,13 @@ bool DashVnetOrch::addPaValidationPost(const string& key, const VnetMapBulkConte
     sai_status_t status = *it_status++;
     if (status != SAI_STATUS_SUCCESS)
     {
-        /* PA validation entry add failed. Remove PA refcount entry */
         pa_refcount_table_.erase(pa_ref_key);
         if (status == SAI_STATUS_ITEM_ALREADY_EXISTS)
         {
-            // Retry if item exists in the bulker
-            return false;
+            return true;
         }
 
+        /* PA validation entry add failed. Remove PA refcount entry */
```

Comment:
Should the comment be above (before erase operation on pa_refcount_table_)?

Reply:
fixed